src/game/camera.c

```diff
@@ -35,6 +35,8 @@ void InitialiseCamera()
     Camera.flags = 0;
     Camera.bounce = 0;
     Camera.number = NO_CAMERA;
+    Camera.additional_angle = 0;
```

Comment:
however you are not overriding it, it still rotates with Lara. 
So the camera angle  is Lara.y_rot + additional_angle

from line 338: `PHD_ANGLE angle = item->pos.y_rot + Camera.target_angle;`

if we override the angle then it would always face the same way.

Reply:
Sorry, I missed that previous message.